openedx/core/djangoapps/theming/template_loaders.py

```diff
@@ -35,6 +35,18 @@ def __init__(self, engine, dirs=None):
             self.dirs = theme_dirs + self.dirs
         super().__init__(engine, self.dirs)
 
+    def get_dirs(self):
+        """
+        Override get_dirs method.
+
+        Make the theme templates a priority, avoiding cashing templates for django ones.
+        """
+        self.dirs = self.engine.dirs
```

Comment:
This code overrides any `dirs` that may have been [passed into the constructor](https://github.com/django/django/blob/main/django/template/loaders/filesystem.py#L13-L15).
Best to let the [parent class get_dirs](https://github.com/django/django/blob/main/django/template/loaders/filesystem.py#L17-L18) help us here:

```suggestion
        dirs = super().get_dirs()
```